pronto_mia/ios/Runner/Info.plist

```diff
@@ -17,13 +17,15 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(FLUTTER_BUILD_NAME)</string>
+	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(FLUTTER_BUILD_NUMBER)</string>
+	<string>$(CURRENT_PROJECT_VERSION)</string>
```

Comment:
Why do those values have to be different?

Reply:
> Please also remove MAC-Specific DS_Store files.

Removed the files and moved .gitignore to root directory.

Reply:
I am still not sure why this values have to differ from the flutter_build